### DISCUSSION TITLE
Where are generated images stored?

### DISCUSSION THREAD

**Discussion by A1ex-N** [painebenjamin/app.enfugue.ai#95] [Q&A] [answered]
I want to know where generated images are stored on disc. For example when you go to File > History and it shows your history. Where is it getting those images from?

**Comment by painebenjamin** [comment, marked as answer]
Hello!

Thank you for using Enfugue! The images are generated at `~/.cache/enfugue/images`, where `~` is your home directory (`/home/<username>` on Linux/MacOS or `C:\Users\<username>` on Windows.)

This was chosen to follow HuggingFace's file location, but I've been made aware this is confusing for users who are not using to dotfiles. I'd welcome anyone's feedback or suggestions on a better default location.

As of `v0.3`, I'm adding the images and intermediates directory to the directory initialization form:
![image](https://github.com/painebenjamin/app.enfugue.ai/assets/57536852/4b6fb565-3f0f-4cbb-a779-84c075184e92)

They will also be changeable in the "Installation Directory" menu:
![image](https://github.com/painebenjamin/app.enfugue.ai/assets/57536852/03ce3f95-05c6-4e3b-a645-f49232aa560e)

The scheduled release for this update 11/03/23. A brief note, any file in the "intermediates" directory will be deleted one hour after it's last modification time - it is where in-progress samples are stored.

Thank you again!
